The upgrade is not accentuated letters proof

When running the upgrade using c2ctool, I encountered issues with accents.
For example, an accentuated letter in `project.yaml.mako` for `mobile_application_title` raises encoding/codec issues.

Here it is:
```
Traceback (most recent call last):
  File ".build/venv/bin/pcreate", line 9, in <module>
    load_entry_point('pyramid==1.6.dev0', 'console_scripts', 'pcreate')()
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scripts/pcreate.py", line 16, in main
    return command.run()
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scripts/pcreate.py", line 85, in run
    return self.render_scaffolds()
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scripts/pcreate.py", line 124, in render_scaffolds
    scaffold.run(self, output_dir, vars)
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scaffolds/template.py", line 65, in run
    self.write_files(command, output_dir, vars)
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scaffolds/template.py", line 97, in write_files
    template_renderer=self.render_template,
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scaffolds/copydir.py", line 84, in copy_dir
    dest_full = os.path.join(dest, substitute_filename(name, vars))
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/pyramid/scaffolds/copydir.py", line 253, in substitute_filename
    fn = fn.replace('+%s+' % var, str(value))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 5: ordinal not in range(128)
Traceback (most recent call last):
  File ".build/venv/bin/c2ctool", line 9, in <module>
    load_entry_point('c2cgeoportal==1.6.4', 'console_scripts', 'c2ctool')()
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 101, in main
    c2ctool.upgrade()
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 213, in upgrade
    print(self.color_bar)
  File "/home/pierre/gr_webmap/.build/venv/local/lib/python2.7/site-packages/c2cgeoportal/scripts/c2ctool.py", line 330, in step1
    ])
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['.build/venv/bin/pcreate', '--overwrite', '--scaffold=c2cgeoportal_update', '../gr_webmap', 'package=gr_mapfish3']' returned non-zero exit status 1
make: *** [upgrade1] Error 1
```

--- a/c2cgeoportal/scaffolds/__init__.py
+++ b/c2cgeoportal/scaffolds/__init__.py
@@ -162,6 +162,6 @@ class TemplateUpdate(BaseTemplate):  # pragma: no cover
             project = load(file("project.yaml", "r"))
             if "template_vars" in project:
                 for key, value in project["template_vars"].items():
-                    vars[key] = value
+                    vars[key] = value.encode("utf-8")
 
         return BaseTemplate.pre(self, command, output_dir, vars)

